ontology/yaml/resources/HVAC/entity_types/FAN.yaml

```diff
@@ -477,6 +477,14 @@ FAN_EFHLC_COCDSP_NOCDSP:
   - COCDSP
   - NOCDSP
 
+FAN_EFVSC_SCRSS:
+  description: "Kitchen exhaust fan with scrubber start/stop and status."
```

Comment:
Kitchen exhaust are not required as part of the description, since nothing on this device need indicate that it is an exhuast fan, nor that it serves a kitche. the same type can be applied anywhere. Perhaps an update for description could be:

"Fan with scrubber and variable speed monitoring (such as for a kitchen exhaust fan)."

This indicates where it might be seen that you have the scrubber on a fan, but doesnt necessitate that all future uses of this type be used for kitchen exhaust fans exclusively.

Reply:
Modified